gulpfile.ts

```diff
@@ -61,13 +63,11 @@ function spawn(command, args, options) {
         } else if (fse.pathExistsSync(command + '.cmd')) {
             command = command + '.cmd';
         }
-
     }
 
     return cp.spawn(command, args, options);
 }
 
-function preWebpack() {
-    // without this, webpack can run out of memory in some environments
-    env.NODE_OPTIONS = '--max-old-space-size=8192';
-}
+exports['webpack-dev'] = () => webpack('development');
```

Comment:
How about using export function test() etc?

Reply:
I'm mostly following the pattern in [the docs](https://github.com/gulpjs/gulp/blob/master/docs/getting-started/3-creating-tasks.md). I think they do this because tasks are often the combination of multiple functions using `gulp.series` and `gulp.parallel` rather than just a single function. Also you can't name a function with a hyphen and I don't feel like refactoring the `webpack-*` task names.

I'm not opposed, but not gonna do it right now.